scripts/verify/lib/VibeOfflineSkillsAudit.ps1

```
@@ -225,54 +225,75 @@ function Get-VibeOfflineCanonicalSkillMap {
-                [void]$requiredSet.Add([string]$candidate)
+    if ($null -eq $inventory -or $null -eq $inventory.public_entry_skills) {
+        $defaultProfile = ""
+        if ($Packaging.PSObject.Properties.Name -contains "default_profile") {
+            $defaultProfile = [string]$Packaging.default_profile
+        }
+        if (
+            -not [string]::IsNullOrWhiteSpace($defaultProfile) -and
+            $Packaging.PSObject.Properties.Name -contains "profiles" -and
+            $Packaging.profiles.PSObject.Properties.Name -contains $defaultProfile
+        ) {
+            $profile = $Packaging.profiles.$defaultProfile
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'profile' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)